### PR TITLE
fix(gitlab) Improve instructions and fix back button

### LINKED ISSUE #17200: Button "back to instructions" doesn't work - clicking it does nothing
image

### LINKED ISSUE #17202: Unclear instructions for GitLab integration setup
> Step 1: Create a Sentry App in GitLab

> To configure GitLab with Sentry, you will need to create a Sentry app in your GitLab instance.

> You'll also need to be a maintainer or owner in GitLab. Projects owned by users are not supported.

> Navigate to the **User Settings** section of your GitLab instance.

> In the sidebar, select 'Applications'.

Did the author mean `https://gitlaburl/admin`? If so, this is called `Admin Area`:

image

Also, providing a direct URL won't hurt. Why not something like this:

> Navigate to the Admin Area of your GitLab instance, then select "Applications" in the sidebar. (Or navigate directly to /admin/applications/new)

### PR DESCRIPTION
**Pull request by markstory**
Fixes #17200
Fixes #17202

### CODE REVIEW

File: src/sentry/pipeline/__init__.py [hunk 1]
```diff
@@ -216,6 +216,8 @@ def current_step(self):
             return self.finish_pipeline()
 
         step = self.pipeline_views[step_index]
+        import logging
+        logging.info('current step %s', step_index)
```
Comment (dashed) [inline src/sentry/pipeline/__init__.py]: stray logging?

Comment (dashed) [inline src/sentry/pipeline/__init__.py]: stray logging?

Comment (markstory) [reply src/sentry/pipeline/__init__.py]: Whoops

**Review by dashed — APPROVED** [review]
Looks good to me 👍 I tagged ecosystem folks for a heads up.

**Pull request merged**